sdk/template-LLC/Azure.Template.LLC/src/properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+// Replace with test project/test project public key and uncomment to make internal members visible to
+// your test project. If not needed, this can be deleted.
+// [assembly: InternalsVisibleTo("Azure.Template.LLC.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
+
+// Replace Microsoft.Test with the correct resource provider namepace for your service and uncomment.
+// See https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers
+// for the list of possible values.
+[assembly: Azure.Core.AzureResourceProviderNamespace("Microsoft.Test")]
```

Comment:
Do we need this?

Reply:
We have these files in a template project, so that people can just copy and paste this project. I'll add the details in LLC Quick doc for the things that needs to be updated.